README.md

```diff
@@ -348,7 +348,7 @@ aborted.
 
 ### Manual Redirect
 
-Since it is not possible to manually follow an HTTP redirect on server-side,
+Since it is not possible to manually follow an HTTP redirect server-side,
```

Comment:
Hm, not sure

Reply:
Could be "on the server-side"?

Reply:
Adjusted in https://github.com/nodejs/undici/pull/1354/commits/4d9f60dd622662e6fc11b26a19d480cf17b4fc53